.github/workflows/chachacha.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - chachacha
+  pull_request: # Temporary! Only for this chachacha PR testing
+    branches: * # Temporary! Only for this chachacha PR testing
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Publish
+        uses: appleboy/ssh-action@master
+        with:
+          host: 45.33.117.205
+          key: ${{ secrets.CHACHACHA_VM_PRIVATE_KEY }}
+          passphrase: ${{ secrets.CHACHACHA_VM_PASSWORD }}
```

Comment:
It is necessary to provide CHACHACHA_VM_PRIVATE_KEY and CHACHACHA_VM_PASSWORD secrets

Reply:
secret has been added for the `ci` user on the vm